text2speech.py

```diff
@@ -1,9 +1,10 @@
-#!/Users/kaz/anaconda3/bin/python
+#!/usr/bin/python3
```

Comment:
Master has been updated to use the following line, which is the best option thus far.

#!/usr/bin/env python